lib/msal-browser/src/request/SilentRequest.ts

```diff
@@ -47,4 +47,5 @@ export type SilentRequest = Omit<
     prompt?: string;
     state?: string;
     tokenBodyParameters?: StringDict;
+    nonce?: string;
```

Comment:
What's the behavior when a token is served from cache?

Reply:
+1, should nonce be added as a lookup filter?

Reply:
Might need to be treated as a force-iframe refresh as it only gets passed to the /authorize endpoint. @jo-arroyo What was the scenario we're trying to unblock with this?

Reply:
@tnorling This is being used in the benchmark tests. When we last talked about this, I was told not being able to pass this in SilentRequest was an oversight?